sdks/python/container/boot.go

```diff
@@ -105,18 +107,57 @@ func main() {
 
 	// (2) Retrieve and install the staged packages.
 
-	dir := filepath.Join(*semiPersistDir, "staged")
+	func() {
 
-	files, err := artifact.Materialize(ctx, *artifactEndpoint, info.GetRetrievalToken(), dir)
-	if err != nil {
-		log.Fatalf("Failed to retrieve staged files: %v", err)
-	}
+        installCompleteFile := filepath.Join(os.TempDir(), "beam.install.complete")
```

Comment:
The indentation here is wonky (it's using spaces instead of tabs). Please run go fmt on this file.
https://blog.golang.org/go-fmt-your-code

Reply:
Done! Can this be accomplished through the gradle plugin also (w/o extra go install)? Would be nice to have this integrated like Spotless for Java.